src/common/variables.c

```diff
@@ -555,15 +608,32 @@ static char *_get_base_value(dt_variables_params_t *params, char **variable)
   else if(_has_prefix(variable, "SEQUENCE"))
   {
     uint8_t nb_digit = 4;
+    guint shift = 1;
+    // old $(SEQUENCEn) syntax for backwards compatibility
     if(g_ascii_isdigit(*variable[0]))
     {
       nb_digit = (uint8_t)*variable[0] & 0b1111;
       (*variable) ++;
     }
+    // new $(SEQUENCE[n,m]) syntax
+    // allows \[[0-9]+,[0-9]+] (PCER)
+    // everything else will be ignored
+    else if(*variable[0] == '[') 
+    {
+      char *parameters[2] = {NULL};
+      int num = _get_parameters(variable, parameters, 2);
+      if(num == 2 && _is_number(parameters[0]) && _is_number(parameters[1]))
```

Comment:
I would allow having only the first parameter, that is:

`$(SEQUENCE[3])` should be supported with shift being set by default to 0.

Reply:
:+1:  default is 1 so counting starts at 1, not 0.